src/client/Renderer/Renderer.js

```diff
@@ -60,16 +60,28 @@ class Renderer{
         const scene = new THREE.Scene();
 
         const axesHelper = new THREE.AxesHelper(100000);
-
         const gridHelper = new THREE.GridHelper(10, 10);
         gridHelper.translateY(-0.01);
+        gridHelper.name = "GridHelper";
 
         scene.add(gridHelper);
         scene.add(axesHelper);
 
         return scene;
     }
 
+    addGridHelper() {
```

Comment:
O kur jį panaudoja? 
Prie tos pačios temos įjungiant 2D taip pat reiktų this.controls.enableRotate = false;